test/src/test/java/hudson/console/ConsoleAnnotatorTest.java

```diff
@@ -219,7 +219,7 @@ public ConsoleAnnotator newInstance(Object context) {
         @Override
         public ConsoleAnnotator annotate(Object build, MarkupText text) {
             if (text.getText().startsWith("line"))
-                text.addMarkup(0,5,"<b tag="+(n++)+">","</b>");
+                text.addMarkup(0,5,"<b tag="+n+++">","</b>");
```

Comment:
Really? More readable? At least do 
```suggestion
                text.addMarkup(0, 5, "<b tag=" + n++ + ">", "</b>");
```

(But I still prefer to keep parens around these)

Reply:
I think this is more a reflection of Kohsuke's unique coding style than anything else…